packages/formspree-react/src/useForm.ts

```diff
@@ -94,15 +94,15 @@ const useForm = (
       return new FormData(form);
     };
 
-    let formData = isEvent(submissionData)
-      ? await getFormData(submissionData)
+    const formData = isEvent(submissionData)
+      ? getFormData(submissionData)
       : submissionData;
 
     const appendExtraData = (prop: string, value: string) => {
       if (formData instanceof FormData) {
         formData.append(prop, value);
       } else {
-        formData = Object.assign(formData, { [prop]: value });
```

Comment:
`Object.assign` mutates the source object, so assigning it to itself doesn't do anything. It is just unnecessarily convoluted; changed it to something straight-forward.